mobsf/MobSF/utils.py

```diff
@@ -606,3 +606,15 @@ def disable_print():
 # Restore
 def enable_print():
     sys.stdout = sys.__stdout__
+
+
+def get_jadx_timeout_value():
```

Comment:
This function is not going to be reused, Can we not have a function here?

Reply:
Removed, thanks.